website/docs/guides/docs/versioning.mdx

```diff
@@ -145,6 +145,37 @@ versioned_sidebars/version-1.0.0-sidebars.json
 </Tabs>
 ```
 
+:::tip
+
+Versioned sidebar files are, like standard sidebar files plugin root relative - so for the example above your versioned sidebar file may look like:
```

Comment:
```suggestion
Versioned sidebar files are, like standard sidebar files, relative to the content root for the given version — so for the example above, your versioned sidebar file may look like:
```

(Docs plugins use both `/docs` and `/versioned_docs` so there's not a single root to be "relative to".)